### PR TITLE
[FEAT] Create the new 'Blog' page

### PR DESCRIPTION
**Pull request by csouchet**
Covers #187 

Accessible from : http://localhost:8000/blog and https://process-analytics-process-analytics-dev-site_preview-pr-224.surge.sh/blog

All page titles are updated and follow 'PAGE_NAME - Process Analytics', except the **Home** page which keeps 'Process Analytics'.

### CODE REVIEW

File: src/theme/components/PageHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  Image,
+  Button,
+} from 'rebass/styled-components';
+import styled from 'styled-components';
+import { header } from '../../content/HeaderContent';
+import { PAGE, SECTION } from '../utils/constants';
+import Link from './Link';
+import { capitalize } from '../utils/string';
+
+const PageHeader = (): JSX.Element => {
```
Comment (csouchet) [inline src/theme/components/PageHeader.tsx:31]: Almost duplicated from the Header class, but we should replace the Header in the home page & have the same in all pages later.

**Comment by csouchet** [comment]
## Mobile format
### Margin 5
image

### Margin 4
image

## Desktop format
image

**Comment by tbouffard** [comment]
⚠️ I have noticed the following changes
The title of the home page switch from 'Process Analytics' to 'Process Analytics - home'. Impact on Google Analytics (page traffic followup).
All page titles follow 'Process Analytics - PAGE_NAME' we should review this.

File: src/theme/components/PageHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+            .map(id => (
+              <Box
+                key={id}
+                ml={[2, 3]}
+                mt="auto"
+                mb="auto"
+                color="background"
+                fontSize={[2, 3]}
+              >
+                <Link href={`#${id}`} tabIndex={0}>
```
Comment (tbouffard) [inline src/theme/components/PageHeader.tsx:66]: ⚠️ doesn't target the page but an anchor in the page that doesn't exist here. This was working for the home page, but not here.
I guess this should be replaced by `href={`/${id}`}`

File: src/theme/components/PageHeader.tsx (hunk 1 above)
Comment (tbouffard) [reply src/theme/components/PageHeader.tsx:31]: Ok to me, we will see later when we will decide how we manage the header on all pages

**Review by tbouffard — APPROVED** [review]

**Pull request merged**